src/Listener/CodeCoverageListener.php

```diff
@@ -14,13 +14,18 @@
 
 namespace FriendsOfPhpSpec\PhpSpec\CodeCoverage\Listener;
 
+use FriendsOfPhpSpec\PhpSpec\CodeCoverage\Exception\ConfigurationException;
 use PhpSpec\Console\ConsoleIO;
 use PhpSpec\Event\ExampleEvent;
 use PhpSpec\Event\SuiteEvent;
 use SebastianBergmann\CodeCoverage\CodeCoverage;
 use SebastianBergmann\CodeCoverage\Report;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
+use function gettype;
+use function is_array;
+use function is_string;
+
```

Comment:
Are those imports really required? Is this something PHPStan complained about?

Reply:
This is something added automatically by the grumphp code not by me